genMatrix.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
```

Comment:
add `use strict`?

Very much a fan of using JS instead if bash scripts - way easier for most people here to maintain I'd assume 🙂